### ISSUE TITLE
Issues about invalid model

### ISSUE THREAD

**Issue by merlinsun** [bitwuzla/bitwuzla#27]
Hi, for this instance, bitwuzla https://github.com/bitwuzla/bitwuzla/commit/f834fc4909c232e695c036c42eb98f9ac26564d1 (debug build) gives an invalid model.
```
$ bitwuzla delta.smt2
[bzlachkmodel] bzla_check_model: invalid model
$ cat delta.smt2
(define-sort FPN () Float64)
(declare-fun e () FPN)
(assert (exists ((E FPN)) (not (= (fp.fma roundTowardNegative e e e) (fp.fma roundTowardNegative e e (fp.fma roundTowardNegative e e e))))))
(check-sat)
```

OS: Ubuntu 20.04

**Comment by merlinsun** [comment]
Another instance related to it:
```
$ bitwuzla small.smt2
[bzlachkmodel] bzla_check_model: invalid model
$ cat small.smt2
(define-sort FPN () (_ FloatingPoint 2 6))
(declare-fun e () FPN)
(declare-fun x () FPN)
(assert (and (= x (fp.fma roundTowardNegative x x e)) (= (fp.fma roundTowardNegative x x x) (fp.fma roundTowardNegative x x x))))
(check-sat)
```

**Comment by mpreiner** [comment]
Fixed with [56bd426](https://github.com/bitwuzla/bitwuzla/commit/56bd426ca07667e0c6ef5450691f85eed53a41db)
